src/scripts/components/Tooltip.jsx

```diff
@@ -196,7 +215,15 @@ class Tooltip extends React.Component {
    * @return {void}
    */
   closeTooltip() {
+    // Cancel delayed tooltip open if user hovers out of target. On 'mouseleave'.
+    if (this.tooltipTimer) {
+      clearInterval(this.tooltipTimer);
```

Comment:
Technically, you _can_ use `clearInterval` here, but consider using `clearTimeout` for clarity.